flake.nix

```diff
@@ -142,6 +142,11 @@
                   ./scripts/binary_patch_version.sh
                 } ./usr/bin/cometa ${versionFull}
                 ${pkgs.fpm}/bin/fpm -s dir -t deb --name ${pkg.pname} -v ${version} --deb-use-file-permissions usr
+
```

Comment:
may be move script to `rollup-bridge-contracts/scropts`

Reply:
postrm is a script, that is executed, when a deb-package is removed.
It is something common for the whole bundle.
Right now, there is indeed a code only related to the rollup-bridge-contracts...
I would still keep it here

Reply:
if we'll not use this script for clean up another components, better move to the separate folder.